src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getShape().getDescription().toLowerCase()
+                + ", area: " + String.format("%.2f", getArea()) + " sq.units, "
+                + getSizeDescription() + ", color: " + getColor().name().toLowerCase());
+    }
+
+    public abstract double getArea();
```

Comment:
Please check common mistake regarding behaviour and introduce interface for calculating are method
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-abstract-classes-to-set-behaviour-for-classes

Reply:
Agree. 
AreaCalculator with calculateArea() method added.
Thank you!